src/v0/destinations/ga/networkHandler.js

```diff
@@ -2,9 +2,30 @@ const {
   REFRESH_TOKEN
 } = require("../../../adapters/networkhandler/authConstants");
 const {
-  processAxiosResponse
+  processAxiosResponse,
+  getDynamicErrorType
 } = require("../../../adapters/utils/networkUtils");
-const ErrorBuilder = require("../../util/error");
+
+const { RetryableError, NetworkError } = require("../../util/errorTypes");
+const tags = require("../../util/tags");
+
+/**
+ * This function helps to determine type of error occured. According to the response
+ * we set authErrorCategory to take decision if we need to refresh the access_token
+ * or need to disable the destination.
+ * @param {*} code
+ * @param {*} response
+ * @returns
+ */
+const getAuthErrCategory = (code, response) => {
+  switch (code) {
+    case 401:
+      if (!response.error?.details) return REFRESH_TOKEN;
+      return "";
+    default:
+      return "";
+  }
+};
```

Comment:
Lets not introduce drastic changes right now

Reply:
@ujjwal-ab where did this info come from?

Reply:
We are using this in many destinations to get `AuthErrCategory`.

Reply:
Lets not make any drastic changes

Reply:
That's not how you are supposed to interpret the status code. Revert the changes for now and set the hardcoded auth err category like it was before.